src/main/java/core/basesyntax/dao/BetDaoImpl.java

```diff
@@ -1,9 +1,11 @@
 package core.basesyntax.dao;
 
 import core.basesyntax.db.Storage;
+import core.basesyntax.lib.Dao;
 import core.basesyntax.model.Bet;
 import java.util.List;
 
+@Dao
 public class BetDaoImpl implements BetDao {
```

Comment:
**public class BetDaoImp implements BetDao - it's wrong
public class BetDaoImpl implements BetDao - it's right
(KseniiaMakarova said)**
I fixed that